tools/eslint-rules/prefer-common-mustnotcall.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @fileoverview Prefer common.mustNotCall(msg)
```

Comment:
(Ah, I see it's also in the message supplied to the user. Might not be terrible to put it in the comment overview too, though.)